openprovider.php

```diff
@@ -2274,6 +2274,96 @@ public function getServiceDomain($service): string
         return $this->getServiceName($service);
     }
 
+    /**
+     * Gets a list of name server data associated with a domain
+     *
+     * @param string $domain The domain to lookup
+     * @param int $module_row_id The ID of the module row to fetch for the current module
+     * @return array A list of name servers, each with the following fields:
+     *
+     *  - url The URL of the name server
+     *  - ips A list of IPs for the name server
+     */
+    public function getDomainNameServers($domain, $module_row_id = null)
```

Comment:
It seems this method is not fully working, The most recent update of the Domains plugin calls this method to fill the name servers fields in the Manage view, but it does not appear to work:
![Screenshot 2023-11-20 at 3 12 56 PM](https://github.com/blesta/module-openprovider/assets/23648083/cf404e16-c706-4638-a48a-bcd2ebd27e10)

Reply:
Did you set nameservers?  It works for me
![openprovider-nameservers](https://github.com/blesta/module-openprovider/assets/18198499/bdf27dce-7e6e-440d-b499-12660ff0ab05)